spec/components/schemas.yaml

```diff
@@ -891,10 +869,10 @@ components:
           type: string
         links:
           items:
-            $ref: "#/components/schemas/link"
+            $ref: "./schemas.yaml#/components/schemas/link"
           type: array
         stac_version:
-          $ref: "#/components/schemas/stac_version"
+          $ref: "./schemas.yaml#/components/schemas/stac_version"
```

Comment:
Two lines below (line 877) from here and also elsewhere in the spec-code there's still some (copy&paste I guess) references to a sample address in Bonn. Probably we should replace them everywhere in the spec-code with "Musterstrasse 1 in Berne" or similar. :wink:

Reply:
I don't mind having an example in Deutschland and I think this is not related to this PR which is more of reorganizing files in order to make use of IDE code navigation. But feel free to change it in the completion of the validation of the openapi schema.